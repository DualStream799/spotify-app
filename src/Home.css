.Home {
  height: 100vh;
  position: fixed;
  width: 25vw;
  background: linear-gradient(#454d4b, #8f9b97);
  flex-grow: row;

  /* background-color:#8f9b97; */

  /* position:fixed  */

  /* background-color:#161616f5; */
  /* border-top-right-radius: 100px;
    border-bottom-right-radius: 100px; */
}

.player {
  width: 100%;
  height: 80px;
}


.follow_artist {
    width: 16.5vw;
    height: 56px;
    border: none;
    overflow: hidden; 
}

.lado{
    display: flex;
    flex-direction: row; 
    margin-left: 30px;
   
}

.lado2{
    display: flex;
    align-items: space-between;
    flex-direction: row; 
    margin-left: 50px;
    margin-top: 10px;
    
    
}

.bloco{
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    /* margin-left: 250px; */
    margin-left: 45vh;
    /* margin-top: -35px; */
    padding-top: 30px;
    max-width: 100%;
    width: 100%;
    color:white;
    background-color: black;
}

.bloco-favoritos{   
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    /* margin-left: 250px; */
    margin-left: 45vh;
    margin-top: -35px;
    padding-top: 30px;
    max-width: 100%;
    color:white;
    background-color: black;
    width: 87.19%;

}

.bloco3 {
  display: flex;
  align-items: center;
  font-size: 22px;
  font-weight: bold;
  margin-left: 280px;
  justify-content: center;
  font-family: Arial, Helvetica, sans-serif;
  font-weight: bold;
  padding-top: 30px;
  max-width: 100%;
  width: 100%;
  color: white;
  min-width: 100%;
  margin-top: 300px;
}

.Home {
  height: 100vh;
  position: fixed;
  width: 25vw;
  background: linear-gradient(#454d4b, #8f9b97);
  flex-grow: row;
}

.player {
  width: 100%;
  height: 80px;
}

.follow_artist {
  width: 16.5vw;
  height: 56px;
  border: none;
  overflow: hidden;
}

.lado {
  display: flex;
  flex-direction: row;
  margin-left: 30px;
}

.lado2 {
  display: flex;
  align-items: space-between;
  flex-direction: row;
  margin-left: 50px;
  margin-top: 50px;
}

.bloco {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  /* margin-left: 250px; */
  margin-left: 45vh;
  margin-top: -35px;
  padding-top: 30px;
  max-width: 100%;
  width: 100%;
  color: white;
}

.bloco-favoritos {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  /* margin-left: 250px; */
  margin-left: 45vh;
  margin-top: -35px;
  padding-top: 30px;
  max-width: 100%;
  color: white;
  width: 87.19%;
}

.form_group {
  position: relative;
  padding: 15px 0 0;
  margin-top: 10px;
  width: 50%;
  flex-direction: row;
  display: flex;
  margin-left: 30px;
}

.form_field {
  font-family: inherit;
  width: 100%;
  border: 0;
  border-bottom: 2px solid gray;
  outline: 0;
  font-size: 1.3rem;
  color: white;
  padding: 7px 0;
  background: transparent;
  transition: border-color 0.2s;
}

.bloco3 {
  display: flex;
  align-items: center;
  font-size: 22px;
  font-weight: bold;
  margin-left: 30vh;
  justify-content: center;
  font-family: Arial, Helvetica, sans-serif;
  font-weight: bold;
  padding-top: 30px;
  max-width: 100%;
  width: 100%;
  color: white;
  /* min-width: 100%; */
  margin-top: 300px;
}

.btnSearch {
  background-color: white;
  width: 120px;
  color: black;
  height: 40px;
  border-radius: 18px;
  margin-left: 20px;
  font-size: 15px;
}

.btnPlaylist {
  background-color: #454d4b;
  width: 280px;
  color: white;
  height: 40px;
  border-radius: 18px;
  margin-left: 350px;
  font-size: 15px;
  margin-top: 30px;
  margin-bottom: 10px;
}

.details-right {
  margin-left: 50vh;
}

.body_home {
  display: flex;
  justify-content: flex-start;
  height: 100vh;
  /* width: 100vh; */
}

.link {
  text-decoration: none !important;
  color: white;
}

.img {
  filter: blur(8px);
  -webkit-filter: blur(8px);
}

.botoes {
  text-decoration: none;
  text-decoration-line: none;
  text-decoration-color: #ffffff;
  display: grid;
  grid-template-columns: auto;
  grid-gap: 30px;
  z-index: 2;
  padding-top: 20px;
}

.images {
  display: flex;
  flex-direction: column;
}

.imagesRound {
  display: flex;
  border-radius: 100px;
  align-self: center;
  display: block;
  margin-left: auto;
  margin-right: auto;
}

.btn {
  background-color: transparent;
  border-color: transparent;
  color: #ffffff;
  z-index: 2;
  font-size: 18px;
  text-align: left;
  font-family: Arial, Helvetica, sans-serif;
  font-weight: bold;
}
.btn:focus {
  outline: none !important;
  outline-offset: none !important;
}

.images {
  display: flex;
  flex-direction: column;
}

.imagesRound {
  display: flex;
  /* flex-direction: column; */
  border-radius: 100px;
  align-self: center;
  display: block;
  margin-left: auto;
  margin-right: auto;
}

.btn {
  background-color: transparent;
  border-color: transparent;
  color: #ffffff;
  z-index: 2;
  font-size: 18px;
  text-align: left;
  font-family: Arial, Helvetica, sans-serif;
  font-weight: bold;
}
.btn:focus {
  outline: none !important;
  outline-offset: none !important;
}

.btnSearch:focus {
  outline: none !important;
  outline-offset: none !important;
}

.btnPlaylist:focus {
  outline: none !important;
  outline-offset: none !important;
}

.coluna {
  display: flex;
  flex-direction: column;
  margin-left: 300px;
}

.info {
  color: #ffffff;
  font-size: 30px;
  text-align: flex-start;
  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
  margin-left: 30px;
  margin-bottom: -10px;
  font-weight: bold;
}

.info2 {
  color: #ffffff;
  font-size: 20px;
  text-align: flex-start;
  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
  margin-left: 30px;
  margin-bottom: -15px;
}

.container {
  display: flex;
  color: white;
}

.organizacao {
  align-items: center;
  display: flex;
  flex-direction: row;
  margin-left: 50px;
}

.desc {
  display: flex;
  flex-direction: column;
  color: white;
}

.lineORetorno {
  width: 920px;
  margin: 15px;
  color: #454d4b;
}

.line {
  width: 100%;
  margin: 15px;
  color: #454d4b;
}

.centro {
  display: block;
  align-self: flex-end;
  margin-left: auto;
  margin-right: auto;
}

.title {
  color: #ffffff;
  font-size: 50px;
  font-family: Arial, Helvetica, sans-serif;
  margin-left: 20px;
}

.escutados {
  align-items: flex-start;
  display: flex;
  flex: 1;
  flex-direction: column;
  color: white;
  min-width: 100%;
  width: 100%;
}

.antigas {
  margin-left: 30px;
  font-family: Arial, Helvetica, sans-serif;
  font-size: 18px;
  flex-direction: row;
  display: flex;
  justify-content: space-between;
  margin-right: 50px;
  color: rgb(255, 254, 254);
  width: 800px;
  background-color: rgb(0, 0, 0);
}

.separando {
  flex-direction: row;
  display: flex;
  width: 84%;
  padding-left: 25px;
  color: white;
  justify-content: space-between;
  font-size: 24px;
  margin-bottom: 20px;
  font-weight: bold;
}

.separando_antigas {
  flex-direction: row;
  display: flex;
  width: 74%;
  padding-left: 25px;
  color: white;
  justify-content: space-between;
  font-size: 26px;
  margin-bottom: 20px;
  font-weight: bold;

}

.grid-container {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  align-items: center;
}

.tracks-container {

    display: grid;
    grid-template-columns: 1fr 1fr 1fr; 
    align-items: left;
    width: 74rem;
  }

.musicas{
    flex: 1;
    
    display: flex;
    flex-direction: row;
    justify-content: space-around;
    color:white;
    
}

.grid-item{
    margin: 1vw;
    flex-direction: row;
    color:white;

}

.musicas {
  flex: 1;

  display: flex;
  flex-direction: row;
  justify-content: space-around;
  color: white;
}

.grid-item {
  margin: 1vw;
  flex-direction: row;
  color: white;
}

.TrackName {
  font-weight: bold;
  font-size: 25px;
  margin-bottom: -10px;
  color: white;
}

.containerEscutando {
  display: flex;
  flex-direction: column;
}

.Texto {
  color: white;
}

.centralizacao {
  text-align: center;
}

.blocoEscutando {
  display: flex;
  width: 75vw;
  flex-direction: column;
  align-items: center;
  width: 100vh;
  margin-left: 42px;
}

.musicatxt {
  color: white;
  font-size: 25px;
  font-weight: bold;
  align-self: center;
}


.antigastxt{
    flex-grow: row;
    width: 24rem;
    margin-left: 2rem;
    margin-right: 2rem;
}

.albumtxt {
  color: white;
  font-size: 20px;
  display: flex;
  align-self: center;
  margin-top: -30px;
}

.containerGeral {
  width: 100%;
  margin-left: 250px;
  /* flex:1; */
  /* width:100vw;
    margin-left: 25vw; */
  /* flex:1; */
  display: flex;
  flex-direction: column;
  height: 100%;
  /* min-height: 100%; */
  /* min-width: 100%; */
}

.descricao {
  margin-left: 30px;
  font-size: 18px;
}

.input {
  display: flex;
  height: 40px;
  width: 200px;
  align-items: center;
  border-radius: 20px;
  justify-content: center;
  margin-left: 30px;
  background-color: white;
  margin-bottom: 10px;
}

.lista {
  display: grid;
  grid-template-columns: 2fr 2fr;
}

.artistaTitulo {
  margin-left: 30px;
  font-size: 30px;
}

.inputGenero {
  display: flex;
  height: 40px;
  width: 200px;
  align-items: center;
  border-radius: 20px;
  justify-content: center;
  margin-left: 30px;
  background-color: white;
  margin-bottom: 10px;
}
.generos {
  display: grid;
  grid-template-columns: 4fr 4fr 4fr;
  margin-left: 130px;
}

.blocoFavoritos {
  margin-left: 30px;
}

.testeCheck {
  color: black;
}

.btnSearch {
  align-self: center;
  margin-top: 30px;
  margin-left: 105px;
  font-weight: bold;
  height: 40px;
  color: "#454D4B";
}

.blocoHome {
  align-items: center;
  justify-content: center;
  display: flex;
  margin-left: 480px;
  margin-top: 150px;
  flex-direction: column;
}

label,
label::before {
  -webkit-transition: 0.25s all ease;
  -o-transition: 0.25s all ease;
  transition: 0.25s all ease;
}

input[type="checkbox"]:checked + label,
input[type="radio"]:checked + label {
  padding-left: 1em;
  color: black;
}

.txt {
  display: flex;
  align-items: center;
  flex-direction: column;

  color: rgb(253, 246, 246);
  margin-top: 60px;
  font-size: 22px;
}

.bemVindo {
  font-weight: bold;
  font-size: 24px;
  margin-bottom: -16px;
}

.dark-mode {
  background-color: black;
}

.light-mode {
  background-color: #9da7a4;
}

:root {
  --main-bg-color: rgb(27, 59, 38);
}
