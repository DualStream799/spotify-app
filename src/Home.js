import { BrowserRouter, Switch, Route } from 'react-router-dom';
import NewReleases from './components/NewReleases';
import App from './App'
import Recomended from './components/Recomended'
import Favorites from './components/Favorites';
import Nav from './components/Nav' 
import UserPlaylist from './components/UserPlaylist'
import { Component } from 'react';
import axios from 'axios';

class Home extends Component {
	constructor(props) {
		super(props);
		const parametros = this.getHashParams();
		this.token = parametros.access_token;
		localStorage.setItem('token', this.token)

		this.state = {
		};
	}

	componentDidMount= () => {
		axios
			.get('https://api.spotify.com/v1/me/tracks?limit=30', {
				headers: {
					Authorization: `Bearer ${this.token}`
				}
			})
			.then((response) => {
				this.setState({
					dataTracks: response.data.items,
					data: response.data.items,
				});
				console.log(this.state.data)
			})
			.catch((erro) => console.log(erro.response.data));
	};


	getHashParams = () => {
		var hashParams = {};
		var e,
			r = /([^&;=]+)=?([^&;]*)/g,
			q = window.location.hash.substring(1);
		e = r.exec(q);
		while (e) {
			hashParams[e[1]] = decodeURIComponent(e[2]);
			e = r.exec(q);
		}
		return hashParams;
	}
	render() {
	return(
		<BrowserRouter>
			<div>
				<Nav />
				<Switch>
					<Route exact path='/' component={App}/>
					<Route exact path='/newReleases' component={NewReleases}/>
					<Route exact path='/recomended' component={Recomended}/>
					<Route exact path='/Favorites' component={Favorites}/>
					<Route exact path='/userPlaylist' component={UserPlaylist}/>
				</Switch>
			</div>
		</BrowserRouter>
	)}
}

export default Home;