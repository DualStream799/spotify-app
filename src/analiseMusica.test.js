import React from 'react';
import Home from './Home';
import { mount } from 'enzyme';
import toJson from "enzyme-to-json"
import Select from '@material-ui/core/Select';

it("should render minhas musicas com botÃ£o analise", () => {
    const props = {
        likedTracks: true,
        boolean_analise: false
    }
    const app = mount(<Home {...props} />)
    expect(toJson(app)).toMatchSnapshot()
})
it("should render analise", () => {
    const props = {
        likedTracks: false,
        boolean_analise: true
    }
    const app = mount(<Home {...props} />)
    expect(toJson(app)).toMatchSnapshot()
})

