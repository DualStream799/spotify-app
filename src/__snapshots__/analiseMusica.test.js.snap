// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render analise 1`] = `
<Home
  boolean_analise={true}
  likedTracks={false}
>
  <div
    className="body_home"
  >
    <div
      className="Home"
    >
      <div
        class="img"
      />
      <div
        className="botoes"
      >
        <button
          class="btn"
        >
          <a
            class="link"
            href="https://spotifyapimanager.herokuapp.com/login"
          >
            Logar com Spotify
          </a>
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Suas playlists
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Suas músicas mais ouvidas
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Seus favoritos
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Seus artistas mais ouvidos
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Novos lançamentos
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Minhas Músicas
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Escutando
        </button>
        <div
          className="lado2"
        >
          <a
            href=" https://pt-br.facebook.com/"
          >
            <AiFillFacebook
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-92.4 233.5h-63.9c-50.1 0-59.8 23.8-59.8 58.8v77.1h119.6l-15.6 120.7h-104V912H539.2V602.2H434.9V481.4h104.3v-89c0-103.3 63.1-159.6 155.3-159.6 44.2 0 82.1 3.3 93.2 4.8v107.9z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiFillFacebook>
          </a>
          <a
            href="https://twitter.com/login?lang=pt"
          >
            <AiFillTwitterSquare
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zM727.3 401.7c.3 4.7.3 9.6.3 14.4 0 146.8-111.8 315.9-316.1 315.9-63 0-121.4-18.3-170.6-49.8 9 1 17.6 1.4 26.8 1.4 52 0 99.8-17.6 137.9-47.4-48.8-1-89.8-33-103.8-77 17.1 2.5 32.5 2.5 50.1-2a111 111 0 0 1-88.9-109v-1.4c14.7 8.3 32 13.4 50.1 14.1a111.13 111.13 0 0 1-49.5-92.4c0-20.7 5.4-39.6 15.1-56a315.28 315.28 0 0 0 229 116.1C492 353.1 548.4 292 616.2 292c32 0 60.8 13.4 81.1 35 25.1-4.7 49.1-14.1 70.5-26.7-8.3 25.7-25.7 47.4-48.8 61.1 22.4-2.4 44-8.6 64-17.3-15.1 22.2-34 41.9-55.7 57.6z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiFillTwitterSquare>
          </a>
          <a
            href=" https://www.instagram.com/"
          >
            <AiFillInstagram
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M512 378.7c-73.4 0-133.3 59.9-133.3 133.3S438.6 645.3 512 645.3 645.3 585.4 645.3 512 585.4 378.7 512 378.7zM911.8 512c0-55.2.5-109.9-2.6-165-3.1-64-17.7-120.8-64.5-167.6-46.9-46.9-103.6-61.4-167.6-64.5-55.2-3.1-109.9-2.6-165-2.6-55.2 0-109.9-.5-165 2.6-64 3.1-120.8 17.7-167.6 64.5C132.6 226.3 118.1 283 115 347c-3.1 55.2-2.6 109.9-2.6 165s-.5 109.9 2.6 165c3.1 64 17.7 120.8 64.5 167.6 46.9 46.9 103.6 61.4 167.6 64.5 55.2 3.1 109.9 2.6 165 2.6 55.2 0 109.9.5 165-2.6 64-3.1 120.8-17.7 167.6-64.5 46.9-46.9 61.4-103.6 64.5-167.6 3.2-55.1 2.6-109.8 2.6-165zM512 717.1c-113.5 0-205.1-91.6-205.1-205.1S398.5 306.9 512 306.9 717.1 398.5 717.1 512 625.5 717.1 512 717.1zm213.5-370.7c-26.5 0-47.9-21.4-47.9-47.9s21.4-47.9 47.9-47.9 47.9 21.4 47.9 47.9a47.84 47.84 0 0 1-47.9 47.9z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiFillInstagram>
          </a>
          <a
            href=" https://www.linkedin.com/"
          >
            <AiOutlineLinkedin
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M847.7 112H176.3c-35.5 0-64.3 28.8-64.3 64.3v671.4c0 35.5 28.8 64.3 64.3 64.3h671.4c35.5 0 64.3-28.8 64.3-64.3V176.3c0-35.5-28.8-64.3-64.3-64.3zm0 736c-447.8-.1-671.7-.2-671.7-.3.1-447.8.2-671.7.3-671.7 447.8.1 671.7.2 671.7.3-.1 447.8-.2 671.7-.3 671.7zM230.6 411.9h118.7v381.8H230.6zm59.4-52.2c37.9 0 68.8-30.8 68.8-68.8a68.8 68.8 0 1 0-137.6 0c-.1 38 30.7 68.8 68.8 68.8zm252.3 245.1c0-49.8 9.5-98 71.2-98 60.8 0 61.7 56.9 61.7 101.2v185.7h118.6V584.3c0-102.8-22.2-181.9-142.3-181.9-57.7 0-96.4 31.7-112.3 61.7h-1.6v-52.2H423.7v381.8h118.6V604.8z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiOutlineLinkedin>
          </a>
        </div>
      </div>
    </div>
    <div
      className="blocoHome"
    >
      <div>
        <img
          src="logoLindo.png"
        />
      </div>
      <div
        className="txt"
      >
         
        <h4
          className="bemVindo"
        >
          Bem vindo ao Musics4U!
        </h4>
        <p
          className="homeDesc"
        >
           
          Para ter acesso às suas funcionalidades utilize a navbar ao lado !
           
        </p>
      </div>
    </div>
  </div>
</Home>
`;

exports[`should render minhas musicas com botão analise 1`] = `
<Home
  boolean_analise={false}
  likedTracks={true}
>
  <div
    className="body_home"
  >
    <div
      className="Home"
    >
      <div
        class="img"
      />
      <div
        className="botoes"
      >
        <button
          class="btn"
        >
          <a
            class="link"
            href="https://spotifyapimanager.herokuapp.com/login"
          >
            Logar com Spotify
          </a>
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Suas playlists
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Suas músicas mais ouvidas
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Seus favoritos
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Seus artistas mais ouvidos
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Novos lançamentos
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Minhas Músicas
        </button>
        <button
          class="btn"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "transparent",
            }
          }
        >
          Escutando
        </button>
        <div
          className="lado2"
        >
          <a
            href=" https://pt-br.facebook.com/"
          >
            <AiFillFacebook
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-92.4 233.5h-63.9c-50.1 0-59.8 23.8-59.8 58.8v77.1h119.6l-15.6 120.7h-104V912H539.2V602.2H434.9V481.4h104.3v-89c0-103.3 63.1-159.6 155.3-159.6 44.2 0 82.1 3.3 93.2 4.8v107.9z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiFillFacebook>
          </a>
          <a
            href="https://twitter.com/login?lang=pt"
          >
            <AiFillTwitterSquare
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zM727.3 401.7c.3 4.7.3 9.6.3 14.4 0 146.8-111.8 315.9-316.1 315.9-63 0-121.4-18.3-170.6-49.8 9 1 17.6 1.4 26.8 1.4 52 0 99.8-17.6 137.9-47.4-48.8-1-89.8-33-103.8-77 17.1 2.5 32.5 2.5 50.1-2a111 111 0 0 1-88.9-109v-1.4c14.7 8.3 32 13.4 50.1 14.1a111.13 111.13 0 0 1-49.5-92.4c0-20.7 5.4-39.6 15.1-56a315.28 315.28 0 0 0 229 116.1C492 353.1 548.4 292 616.2 292c32 0 60.8 13.4 81.1 35 25.1-4.7 49.1-14.1 70.5-26.7-8.3 25.7-25.7 47.4-48.8 61.1 22.4-2.4 44-8.6 64-17.3-15.1 22.2-34 41.9-55.7 57.6z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiFillTwitterSquare>
          </a>
          <a
            href=" https://www.instagram.com/"
          >
            <AiFillInstagram
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M512 378.7c-73.4 0-133.3 59.9-133.3 133.3S438.6 645.3 512 645.3 645.3 585.4 645.3 512 585.4 378.7 512 378.7zM911.8 512c0-55.2.5-109.9-2.6-165-3.1-64-17.7-120.8-64.5-167.6-46.9-46.9-103.6-61.4-167.6-64.5-55.2-3.1-109.9-2.6-165-2.6-55.2 0-109.9-.5-165 2.6-64 3.1-120.8 17.7-167.6 64.5C132.6 226.3 118.1 283 115 347c-3.1 55.2-2.6 109.9-2.6 165s-.5 109.9 2.6 165c3.1 64 17.7 120.8 64.5 167.6 46.9 46.9 103.6 61.4 167.6 64.5 55.2 3.1 109.9 2.6 165 2.6 55.2 0 109.9.5 165-2.6 64-3.1 120.8-17.7 167.6-64.5 46.9-46.9 61.4-103.6 64.5-167.6 3.2-55.1 2.6-109.8 2.6-165zM512 717.1c-113.5 0-205.1-91.6-205.1-205.1S398.5 306.9 512 306.9 717.1 398.5 717.1 512 625.5 717.1 512 717.1zm213.5-370.7c-26.5 0-47.9-21.4-47.9-47.9s21.4-47.9 47.9-47.9 47.9 21.4 47.9 47.9a47.84 47.84 0 0 1-47.9 47.9z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiFillInstagram>
          </a>
          <a
            href=" https://www.linkedin.com/"
          >
            <AiOutlineLinkedin
              color="white"
              size={40}
            >
              <IconBase
                attr={
                  Object {
                    "viewBox": "0 0 1024 1024",
                  }
                }
                color="white"
                size={40}
              >
                <svg
                  color="white"
                  fill="currentColor"
                  height={40}
                  size={40}
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": "white",
                    }
                  }
                  viewBox="0 0 1024 1024"
                  width={40}
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M847.7 112H176.3c-35.5 0-64.3 28.8-64.3 64.3v671.4c0 35.5 28.8 64.3 64.3 64.3h671.4c35.5 0 64.3-28.8 64.3-64.3V176.3c0-35.5-28.8-64.3-64.3-64.3zm0 736c-447.8-.1-671.7-.2-671.7-.3.1-447.8.2-671.7.3-671.7 447.8.1 671.7.2 671.7.3-.1 447.8-.2 671.7-.3 671.7zM230.6 411.9h118.7v381.8H230.6zm59.4-52.2c37.9 0 68.8-30.8 68.8-68.8a68.8 68.8 0 1 0-137.6 0c-.1 38 30.7 68.8 68.8 68.8zm252.3 245.1c0-49.8 9.5-98 71.2-98 60.8 0 61.7 56.9 61.7 101.2v185.7h118.6V584.3c0-102.8-22.2-181.9-142.3-181.9-57.7 0-96.4 31.7-112.3 61.7h-1.6v-52.2H423.7v381.8h118.6V604.8z"
                    key="0"
                  />
                </svg>
              </IconBase>
            </AiOutlineLinkedin>
          </a>
        </div>
      </div>
    </div>
    <div
      className="blocoHome"
    >
      <div>
        <img
          src="logoLindo.png"
        />
      </div>
      <div
        className="txt"
      >
         
        <h4
          className="bemVindo"
        >
          Bem vindo ao Musics4U!
        </h4>
        <p
          className="homeDesc"
        >
           
          Para ter acesso às suas funcionalidades utilize a navbar ao lado !
           
        </p>
      </div>
    </div>
  </div>
</Home>
`;
